sre/storage/kvStorage.go

```diff
@@ -0,0 +1,57 @@
+package main
```

Comment:
Okay, looks good. small suggestions.
1. the Storage is an interface, not a kvStorage as the filename suggests right?
Put the Storage interface in a separate file called header.go
2. Map is the kvstorage implementation, so all functions and declarations of map stay in this file.
3. the function main is an implementation of map. We don't really need this as your tests check the functionality of your KV store. in real-life applications, you func main() is usually replaced by an http server func or a cli implementation or any other. So remove main.